resources/views/checkout/partials/address-cards-popup.blade.php

```diff
@@ -0,0 +1,30 @@
+<input type="checkbox" id="popup" class="peer hidden"/>
+<div class="fixed inset-0 opacity-0 transition z-50 flex justify-center items-center pointer-events-none peer-checked:opacity-100 peer-checked:pointer-events-auto">
+    <label class="absolute cursor-pointer inset-0 bg-ct-primary/60" for="popup"></label>
+    <div class="bg-ct-inactive-100 p-3 rounded z-10 relative">
+        <x-rapidez-ct::title class="text-24 p-3">@lang('My addresses')</x-rapidez-ct::title>
```

Comment:
can't use text-24 values, please use one of the ct-title components which are present within the checkout-theme